### ISSUE TITLE
BigQuery ignores identifiers in FROM clause

### ISSUE THREAD

**Issue by yuval-illumex** [sqlparser-rs/sqlparser-rs#889]
tl;dr - SELECT * from d.s.t and SELECT * from \`d.s.t\` are equivalent in BigQuery and should return the same response from sqlparser. 

Let's say I have database d, with schema s and table t.

In Snowflake, the following query will work - SELECT * FROM d.s.t 
However, the following query will fail - SELECT * from "d.s.t" as Snowflake will look for a table called d.s.t

However, in BigQuery:
SELECT * from d.s.t and SELECT * from \`d.s.t\` are equivalent and both work. I agree it's not expected and confusing.

I really want to support this in someway in sqlparser: If the Dialect is BigQuery, ignore the identifiers in the FROM clause. 
(I'm getting the wrong result for SELECT * from \`d.s.t\`, the table is with one value instead of 3).

As I'm not a Rust developer, any guidance with this one? Every manipulation i'm trying to do is already after the words are tokenised, so I don't have a way to ignore the identifier. 

Appreciate your guidance here!

Thank you!

**Comment by MartinNowak** [comment]
Why do you care whether it's parsed as `[Ident("d.s.t")]` vs. `[Ident("d"), Ident("s"), Ident("t")]`.
The method is here
https://github.com/sqlparser-rs/sqlparser-rs/blob/097e7ad56ed09e5757fbf0719d07194aa93521e8/src/parser.rs#L4684-L4695
and can be customized like this
https://github.com/sqlparser-rs/sqlparser-rs/blob/097e7ad56ed09e5757fbf0719d07194aa93521e8/src/parser.rs#L6422-L6426
